src/java_bytecode/Makefile

```diff
@@ -66,11 +66,14 @@ core-models.jar: $(all_classes)
 
 CLEANFILES = java_bytecode$(LIBEXT) core-models.jar converter all_classes
 
-all: | java_core_models.inc java_bytecode$(LIBEXT)
+all: java_bytecode$(LIBEXT)
```

Comment:
Commit can be squashed into https://github.com/cesaro/cbmc/pull/5/commits/068c730d5164434a1b3c5894d57b04e361cef962